astropy/table/table.py

```diff
@@ -2863,18 +2867,18 @@ def from_pandas(cls, dataframe, index=False):
           >>> df = pd.DataFrame({'time': time})
           >>> df['dt'] = dt
           >>> df['x'] = [3., 4.]
-          >>> df
+          >>> with pd.option_context('display.width', 80):
+          ...     print(df)
                   time       dt    x
           0 1998-01-01 00:00:01  3.0
           1 2002-01-01 00:05:00  4.0
 
-          >>> QTable.from_pandas(df)
-          <QTable length=2>
-                    time            dt      x
-                   object         object float64
-          ----------------------- ------ -------
-          1998-01-01T00:00:00.000    1.0     3.0
-          2002-01-01T00:00:00.000  300.0     4.0
+          >>> t = QTable.from_pandas(df)
+          >>> t.pprint(max_lines=-1, max_width=-1)
```

Comment:
I suspect this change for `QTable` output is not necessary, otherwise there would be lots of other docstring examples failing (e.g. look in http://docs.astropy.org/en/latest/table/operations.html in hstack and join sections).

So it seems like pandas is doing something differently from astropy for detecting console width?

Reply:
OK I reverted this part. Thanks for the review!